src/main/java/com/codeless/api/automation/util/ValidationUtil.java

```diff
@@ -0,0 +1,12 @@
+package com.codeless.api.automation.util;
+
+public final class ValidationUtil {
+
+  private ValidationUtil() {
+    throw new RuntimeException();
+  }
+
+  public static int validateMaxResults(int maxResults) {
+    return Math.min(maxResults, 50);
```

Comment:
I would say, getMaxResultsOrDefault.
50 is a lot, let's do it to default, which is 25.

Reply:
Also, call this class something else, for me, validation either throws error or return true or false.

Reply:
renamed to `DefaultValueUtil.getMaxResultsOrDefault(25)`